optimizer.json

```diff
@@ -33,5 +37,11 @@
         "betas": null,
         "epsilon": null,
         "weight_decay": null
+    },
+    "text_encoder_freezing": {
+        "freeze_embeddings": false,
+        "freeze_front_n_layers": null,
+        "freeze_final_layer_norm": true,
+        "method": "drop"
```

Comment:
Think you mentioned on discord, but if this ends up staying make sure to add some documentation for it.

Reply:
yeah "method" will be gone as soon as i figure out which to use and how to reliably resume optimizer state with it